digos-ambassador/Content/Transformations/Species/snake/Teeth.yml

```diff
@@ -0,0 +1,9 @@
+part: Teeth
+species: snake
+description: "Transforms the target's teeth into that of an anthro snake."
+default_base_colour: "white"
+is_nsfw: false
+shift_message: "The upper canine teeth in {@target}'s mouth vibrate and grow larger, forming into two needle-like fangs which flash menacingly before recracting somewhat. The lower canines twist and melt until they form a smaller pair of generally flatter and rounded fangs"
```

Comment:
Last sentence missing end punctuation. Also feels like it trails off a little? Not sure.